mllib/src/main/scala/org/apache/spark/ml/feature/Binarizer.scala

```diff
@@ -31,10 +31,12 @@ import org.apache.spark.sql.types.{DoubleType, StructType}
  * :: Experimental ::
  * Binarize a column of continuous features given a threshold.
  */
+@Since("1.4.0")
 @Experimental
-final class Binarizer(override val uid: String)
+final class Binarizer @Since("1.4.0") (@Since("1.4.0") override val uid: String)
   extends Transformer with HasInputCol with HasOutputCol {
 
+  @Since("1.4.0")
```

Comment:
Hey @mengxr I'm a little confused here because Binarizer wasn't added until 1.4.0.  I did correct the default constructor to 1.2.0 for the original classes from that version (HashingTF, StandardScaler, Tokenizer).